octoprint_prettygcode/static/js/prettygcode.js

```diff
@@ -16,7 +16,7 @@ $(function () {
                 durJobDate = job.file.date;
                 if(viewInitialized && gcodeProxy)
                     {
-                        gcodeProxy.loadGcode('/downloads/files/local/' + curJobName);
+                        gcodeProxy.loadGcode('downloads/files/local/' + curJobName);
```

Comment:
OctoPrint defines the `BASEURL` global variable available to all plugins:

```suggestion
                        gcodeProxy.loadGcode(BASEURL + 'downloads/files/local/' + curJobName);
```

Reply:
I'm their that's what my change should do, no need to add base url for actual relative links.